packages/react/src/components/Accordion/Accordion.js

```diff
@@ -16,13 +16,15 @@ function Accordion({
   children,
   className: customClassName,
   disabled = false,
+  flush = false,
```

Comment:
Hi @jnm2377 👋 
 thanks for taking the time to contribute this! Everything looks really good, I just have one comment. We want our API to be consistent across all our components. Could you update the prop flush to be named isFlush instead so it matches our prop in our StructuredList component here: [https://github.com/carbon-design-system/carbon/blob/a2afe009004bff3c4e417112e03206[…]/packages/react/src/components/StructuredList/StructuredList.js](https://github.com/carbon-design-system/carbon/blob/a2afe009004bff3c4e417112e03206d37fcd9bd7/packages/react/src/components/StructuredList/StructuredList.js#L71-L74)? Thank you! 🙂